CHANGES/10137.misc.rst

```diff
@@ -0,0 +1,3 @@
+Restored support for zero copy writes when using Python 3.12 versions 3.12.9 and later or Python 3.13.2+ -- by :user:`bdraco`.
+
+Zero copy writes were previously disabled due to CVE-2024-12254 which is resolved in these Python versions.
```

Comment:
Sphinx v8.1 (which I've just checked that our RTD setup uses already) added the `:cve:` role (it was previously only available through `sphinx-issues`). So we can now do this:
```suggestion
Zero copy writes were previously disabled due to :cve:`2024-12254` which is resolved in these Python versions.
```

Reply:
nice!

Reply:
/home/runner/work/aiohttp/aiohttp/docs/changes.rst:815: ERROR: Unknown interpreted text role "cve".

Looks like we might need something else?